### PR TITLE
Merge AspNetFullFrameworkSampleApp to master

### PR DESCRIPTION
**Pull request by SergeyKleyman**
Merging Full framework sample from my repo to main/master

### CODE REVIEW

**Comment by SergeyKleyman** [comment]
@kuisathaverat I fixed Jenkinsfile to make Full framework sample work - please take a look to make sure I didn't break anything.

File: Jenkinsfile [hunk 1]
```diff
@@ -72,7 +72,7 @@ pipeline {
                     unstash 'source'
                     dir("${BASE_DIR}"){
                       sh '''
-                      dotnet sln remove sample\\AspNetFullFrameworkSampleApp\\AspNetFullFrameworkSampleApp.csproj
+                      dotnet sln remove sample/AspNetFullFrameworkSampleApp/AspNetFullFrameworkSampleApp.csproj
```
Comment (kuisathaverat) [inline Jenkinsfile]: ouch!! c&p

File: sample/AspNetFullFrameworkSampleApp/MyModule1.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
+﻿using Elastic.Apm.DiagnosticSource;
+using System;
+using System.Web;
+using Elastic.Apm;
+
+namespace AspNetFullFrameworkSampleApp
+{
+	public class MyModule1 : IHttpModule
```
Comment (gregkalapos) [inline sample/AspNetFullFrameworkSampleApp/MyModule1.cs]: @SergeyKleyman I think we renamed this... Are you sure we have all changes in this PR?

Comment (gregkalapos) [reply sample/AspNetFullFrameworkSampleApp/MyModule1.cs]: Yeah, it's different from the other PR: https://github.com/elastic/apm-agent-dotnet/pull/112/files#diff-6fb69fd24db8b4a2903a7427ad18a0e6R6

Comment (SergeyKleyman) [reply sample/AspNetFullFrameworkSampleApp/MyModule1.cs]: Yes, my bad :( I merged all the changes from FullFrameworkSample branch of main repo.

Comment (gregkalapos) [reply sample/AspNetFullFrameworkSampleApp/MyModule1.cs]: 👍

**Pull request merged**